ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -161,6 +161,7 @@ def _router_bgp_vrfs(self) -> dict | None:
                     "redistribute_routes": ["connected"],
                     "eos_cli": get(vrf, "bgp.raw_eos_cli"),
                     "struct_cfg": get(vrf, "bgp.structured_config"),
+                    "evpn_multicast": (get(vrf, "_l3_multicast_enabled") and self._overlay_vtep and self._overlay_evpn),
```

Comment:
if you check for "switch.evpn_multicast" in the vrf logic, you no longer need to test for vtep and evpn here since those checks are already performed.

Reply:
Fixed